components/youcanbook_me/sources/common/booking-base.mjs

```diff
@@ -0,0 +1,28 @@
+import youcanbook_me from "../../youcanbook_me.app.mjs";
+
+export default {
+  props: {
+    youcanbook_me,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 15 * 60,
+      },
+    },
+  },
+  methods: {
+    async emitEvent() { },
+  },
+  hooks: {
+    async deploy() {
+      const bookings = await this.youcanbook_me.getBookings({});
+
+      bookings.forEach(this.emitEvent);
+    },
+  },
+  async run() {
+    const response = await this.youcanbook_me.getBookings({});
```

Comment:
Shouldn't this support pagination?

Reply:
I tried implement the pagination documented, but don't worked!